ament_tools/build_type.py

```diff
@@ -18,7 +18,9 @@
 
 
 class BuildAction(object):
-    """Represents an action to do at build time, either a command or a functor.
+
+    """
```

Comment:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings says: The summary line may be on the same line as the opening quotes or on the next line.

I will undo the unnecessary wraps to keep the diff small.